build.gradle

```diff
@@ -56,8 +56,8 @@ subprojects {
     apply plugin: 'java-test-fixtures'
     apply plugin: 'kotlin'
 
-    sourceCompatibility = "1.6"
-    targetCompatibility = "1.6"
+    sourceCompatibility = "1.8"
```

Comment:
Is that really necessary?

Reply:
Yes for it to work on my development environment, although I must admit it was easier for me to bump the version than adjust my version of java/kotlin/gradle to reflect what is needed. Feel free to put that back to 1.6.